### ISSUE TITLE
[BUG] Accessibility Menu in Options doesn't have a proper name

### ISSUE THREAD

**Issue by RexenSquared** [UZDoom/UZDoom#102]
### UZDoom version

Built From Source

### Which game are you running with UZDoom?

Doom 2 (KEK)

### What Operating System are you using?

Windows 11

### Have you checked that no other similar issue already exists?

- [x] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

This is most likely already known but I didn't see an issue filed yet, so I figure I'll file one. This also applies to the option names as well. (I take it this is placeholder stuff and is the reason the option names don't have proper names yet)
<img width="2560" height="1600" alt="Image" src="https://github.com/user-attachments/assets/0486ca65-e912-4525-bf9e-b813762f3c3f" />

<img width="2560" height="1600" alt="Image" src="https://github.com/user-attachments/assets/54eb2016-ad13-4601-ba9d-ac8b0f1af743" />

My build is based off commit: https://github.com/UZDoom/UZDoom/commit/7ece36a22dbcb24ccaec65e201f8b9cfd342e0ec
